openpgp/armor/armor.go

```diff
@@ -35,6 +35,7 @@ type Block struct {
 	oReader openpgpReader
 }
 
+// ArmorCorrupt is returned if an armor is invalid.
 var ArmorCorrupt error = errors.StructuralError("armor invalid")
```

Comment:
I know the linter says to add comments to every exported symbol, but some of these are completely self-explanatory even from the name - so then the comment naturally feels kinda redundant.